### ISSUE TITLE
Assessment Results

### ISSUE THREAD

**Issue by francepack** [aetzion1/mod_0_skills#1]
@aetzion1 
Overall, nice work here! You've performed well enough on this assessment to merit making some changes to achieve being `technical ready`. Please make the following updates by EOD Sunday:

- [ ] In your first class, if you have a toppings array, also having a toppings amount attribute is redundant, as you can refer to `toppings.length` any time you'd need that information. Try for a different integer attribute, and method that pairs with it!

- [ ] In your second class, `is_mugged` doesn't read as a method. I'd go with a simple verb form here. If you want to be more descriptive, you can, I'd just put the focus on the attribute you are modifying.

Non-actionable feedback:
- Maria has been your mother all this time!? What a shocking twist in your class work, hah!

- I really like seeing specificity in your method names, just keep in mind you likely won't want to build methods, for every possible scenario, for example an individual method for each individual pizza topping you can add. But hey- end of the day, you are in charge of crafting the methods that most meet your needs, so if sausage happens to be something added frequently to your pizzas, this method is a perfectly fine, descriptive method! An `addTopping` method also works- that would just need to take an argument that represents which topping is being added. Again, no changes needed for this note, just something to consider as you keep making classes!

**Comment by aetzion1** [comment]
@francepack 
Thank you for the feedback! I've made those suggested changes and commited them. Some detail below.

* Great point that my toppings_amnt (count) attribute is redundant given I can use toppings.length for my other attribute. I've replaced the former with spice_level!
* I also can see how the is_mugged method, while making for a good story, doesn't tie naturally to a specific attribute. I've modified it to 'lose_tools'.
* I'll endeavor to work on my methods - making them more useful as tools and not so specific if it's truly not necessary. From my Mod1Prework, I'm learning how much more I can build into these.

Thanks!
